bindings/java/mongocrypt/src/main/java/com/mongodb/crypt/capi/CAPI.java

```diff
@@ -567,7 +554,7 @@ public interface mongocrypt_random_fn extends Callback {
      * Retrieve it with @ref mongocrypt_ctx_status
      */
     public static native boolean
-    mongocrypt_ctx_setopt_query_type (mongocrypt_ctx_t ctx, int query_type);
+    mongocrypt_ctx_setopt_query_type (mongocrypt_ctx_t ctx, cstring query_type, int len);
```

Comment:
Any idea why this method has a `len` parameter where none of the other methods that take `cstring` do?

Reply:
No compelling reason. I was spending a lot of time staring at setopt_algorithm, which just happened to be the only other API that takes an explicit length, so it probably primed me to do the same thing when I updated this function.